checkout.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: lesson
+subtitle: Checkout Procedure
+email: training-help@software-carpentry.org
+---
+After you have finished the two-day training class,
+you must go through these steps in order to complete your training and be certified:
+
+1.  Submit a [small change to one of our lessons](#lesson-change).
+2.  Take part in an online [discussion session](#discussion-session).
+3.  Teach a short [demonstration lesson](#demo-lesson) online.
+
+We would like to get everyone through this process within two months of training,
+so please get started as soon as you can.
+Anyone who has not completed within 90 days should contact their trainer,
+or the head of instructor training,
+to discuss reasons and options.
+
+## Part 1: Submit a Small Change to One of Our Lessons
+<a name="lesson-change"></a>
+
+Our lessons are maintained and improved by the people who teach them,
+so this part of the checkout procedure gives you a chance to familiarize yourself
+with the way we manage that.
```

Comment:
suggest wording: "this part of the checkout procedure gives you a chance to familiarize yourself with our collaborative training material development process." (because we talk about *collaborative training material development* all the time; let's just use that terminology)